src/java.base/share/classes/java/util/PriorityQueue.java

```diff
@@ -209,6 +209,27 @@ else if (c instanceof PriorityQueue<?>) {
         }
     }
 
+    /**
+     * Creates a {@code PriorityQueue} containing the elements in the
+     * specified collection. The {@code PriorityQueue} will order the
+     * priority queue's elements elements according to the specified
+     * comparator.
```

Comment:
I know you updated this in response to Alan's comment, but the second sentence now says "elements elements". And repeating "priority queue" sounds somewhat redundant. Suggestion for the second sentence:

> The elements of the new {@code PriorityQueue} will be ordered according to
> the specified comparator.

Reply:
updated, thanks for the suggestion